packages/vscode-extension/src/resources/Strings.json

```diff
@@ -41,6 +41,8 @@
       "deployDescription": "Deploy your Teams app to the provisioned resources in Azure",
       "publishTitle": "Publish to Teams",
       "publishDescription": "Publish to Teams",
+      "cicdGuideTitle": "Go To CI/CD Guide",
```

Comment:
Better align the format with other items, for example, the title for deploy  is 'Deploy to the Cloud', both 'to ' and 'the' start with lowercase

Reply:
Agree, change it soon.

Reply:
According to UX's suggestion, update it to `CI/CD guide`.